### ISSUE TITLE
Undefined property: GuzzleHttp\Psr7\Response::$ErrorStatus

### ISSUE THREAD

**Issue by michabbb** [adamdburton/destiny-2-api-client#6]
```php
$Manifest = $D2->getDestinyManifest();
if ($Manifest->isSuccess()) {
   //do something
}
```

![22-05-2018-12-22-51](https://user-images.githubusercontent.com/3524595/40332922-44a6e65e-5d56-11e8-8002-6ff096b778f1.png)

**Comment by michabbb** [comment]
maybe like this ?

`return $this->getResponse()->getStatusCode() === 200;`

**Comment by adamdburton** [comment]
Should be resolved with #7, thanks @michabbb!
